src/readsbml.jl

```diff
@@ -356,8 +357,11 @@ function get_model(mdl::VPtr)::SBML.Model
             end
             objectives[get_string(o, :Objective_getId)] = Objective(type, flux_objectives)
         end
-        active_objective =
-            get_optional_string(mdl_fbc, :FbcModelPlugin_getActiveObjectiveId)
+        active_objective = let
+            aoi = get_optional_string(mdl_fbc, :FbcModelPlugin_getActiveObjectiveId)
+            # Keep it to nothing if the string is empty
+            isempty(aoi) ? nothing : aoi
+        end
```

Comment:
I wonder if `active_objective` should be `::String` since `FbcModelPlugin_getActiveObjectiveId` returns an empty string, not a null-pointer, when there is no active objective.

Reply:
(solved in slack I guess -- let's make it empty string because it doesn't break anything)